specification/elastic/resource-manager/Microsoft.Elastic/preview/2021-10-01-preview/examples/UpgradableVersions_Get.json

```diff
@@ -0,0 +1,19 @@
+{
+  "parameters": {
+    "api-version": "2021-10-01-preview",
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "resourceGroupName": "myResourceGroup",
+    "monitorName": "myMonitor"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "currentVersion": "7.15.0",
+        "upgradableVersions": [
+          "7.15.1",
+          "7.16.0"
+        ]
+      }
```

Comment:
Since the api is modeled as a collection get, the response should have 'value' properties which is an array of items the rp returns.

Reply:
done